### ISSUE TITLE
New project: MyProject

### ISSUE THREAD

**Issue by robvanderg** [allenai/allennlp-gallery#22]
Project metadata:
```
{
  "title": "Massive Choice, Ample Tasks (MaChAmp):A Toolkit for Multi-task Learning in NLP",
  "authors": [
   {
      "name": "Rob van der Goot",
      "email": "robv@itu.dk",
      "affiliation": "IT University of Copenhagen"
    },
   {
      "name": "Ahmet Üstün",
      "email": "a.ustun@rug.nl",
      "affiliation": "University of Groningen"
    },
   {
      "name": "Alan Ramponi",
      "email": "alan.ramponi@unitn.it",
      "affiliation": "University of Trento"
    },
   {
      "name": "Ibrahim Sharaf",
      "email": "ibrahim.sharaf@factmata.com",
      "affiliation": "Factmata"
    },
   {
      "name": "Barbara Plank",
      "email": "bapl@itu.dk",
      "affiliation": "IT University of Copenhagen"
    }
  ],
  "submission_date": "2021-03-30",
  "github_link": "https://github.com/machamp-nlp/machamp",
  "paper_link": "https://arxiv.org/pdf/2005.14672.pdf",
  "allennlp_version": "1.3",
  "datasets": [
    {
      "name": "GLUE",
      "link": "https://gluebenchmark.com/"
    },
    {
      "name": "UD",
      "link": "https://universaldependencies.org/"
    },
    {
      "name": "WMT14",
      "link": "https://nlp.stanford.edu/projects/nmt/"
    },
    {
      "name": "IWSLT15",
      "link": "https://nlp.stanford.edu/projects/nmt/"
    }
  ],
  "tags": ["MTL", "Multi-task learning", "sequence labeling", "dependency parsing", "text generation", "masked language modeling"]
}
```

Description:

Transfer learning, particularly approaches that combine multi-task learning with pre-trained contextualized embeddings and fine-tuning, have advanced the field of Natural Language Processing tremendously in recent years. In this paper we present MaChAmp, a toolkit for easy fine-tuning of contextualized embeddings in multi-task settings. The benefits of MaChAmp are its flexible configuration options, and the support of a variety of natural language processing tasks in a uniform toolkit, from text classification and sequence labeling to dependency parsing, masked language modeling, and text generation.

**Comment by dirkgr** [comment]
I would have loved to have known about this before we designed the multi-task system in AllenNLP 2.0. I hope we covered your use case adequately!

**Comment by dirkgr** [comment]
@robvanderg, the title in the JSON says "Challenges in Annotating and Parsing Spoken, Code-switched, Frisian-Dutch Data", but the paper is "Massive Choice, Ample Tasks (MACHAMP): A Toolkit for Multi-task Learning in NLP"?

**Comment by robvanderg** [comment]
Sorry, fixed!

**Comment by robvanderg** [comment]
> I would have loved to have known about this before we designed the multi-task system in AllenNLP 2.0. I hope we covered your use case adequately!

Shows the usefulness of this gallery ;)  . We do many things in a "non-AllenNLP" way though. We also have some papers/projects using MaChAmp (and thus AllenNLP), would it be relevant to add those as well? (see the website for a list: https://machamp-nlp.github.io/)

**Comment by dirkgr** [comment]
Yes, we would love to have all of those in the gallery! Give that "language" field a workout :-)

I love that you're doing work in Frisian. I am the first in my family to not speak Frisian anymore, though I can understand a lot of it when my parents and grandparents speak it.
